spec/providers/coreIntegration/environment.integration.src.js

```diff
@@ -32,4 +32,26 @@ module.exports = function (setup) {
       done();
     });
   });
+
+  it("exposes dynamic require functionality", function (done) {
+    moduleEnv.testRequire(true).then(function (ret) {
+      expect(ret).toEqual(true);
+      done();
+    }, function (ret) {
+      expect(ret).toEqual(true);
+      expect(true).toEqual(false);
+      done();
+    });
+  });
+
+  it("hanles dynamic require failures", function (done) {
```

Comment:
See comment above. Once we have more descriptive results returned, we can assert against those, instead of true/false